src/Disciplina/DB/DSL/Types.hs

```diff
@@ -0,0 +1,100 @@
+{-# LANGUAGE TypeOperators #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE GADTs #-}
+
+module Disciplina.DB.DSL.Types
+       ( Query (..)
+        ,WHERE(..)
+        ,Tx(..)
+        ,Txs(..)
+        ,Obj(..)
+        ,FilterEntity(..)
+        ,FilterExpr(..)
+        ,NodeWithProp(..)
```

Comment:
Can we have spaces after `,`?

Reply:
yes sure, I will add space. @Heimdell only purpose of `WHERE` is to make queries easier to read, as in `SELECE Tx WHERE ...`. Do you think we should remove `WHERE`?